Makefile

```
@@ -227,3 +230,18 @@ release-notes: ## greps UNRELEASED notes from the CHANGELOG
 .PHONY: help
 help: ## this help
 	@ awk 'BEGIN {FS = ":.*##"; printf "Usage: make \033[36m<target>\033[0m\n\nTargets:\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-10s\033[0m\t%s\n", $$1, $$2 }' $(MAKEFILE_LIST) | column -s$$'\t' -t
+
+.PHONY: generate-alerting-client
+generate-alerting-client:
```

This should add the target with the description to the output of `help` target
```suggestion
generate-alerting-client: ## generate Kibana client
```